audio_channels: 1
audio_format: pcm_16
audio_sample_rate: 16000
confidence_threshold: 0.6
cooldown_period: 2.0
enable_asr_detection: true
enable_fallback_detection: true
energy_threshold: 1000
max_audio_length: 5.0
min_audio_length: 0.5
min_segment_length: 0.3
silence_threshold: 0.1
threshold: 0.7
wake_word: 傻强
wake_word_variants:
- 傻强
- 傻强呀
- 傻强啊
- 傻强仔
- 阿强
- 强仔
